### ISSUE TITLE
Document annotations in "Code files"

### ISSUE THREAD

**Issue by TWiStErRob** [ebelinski/apilevels#12]
Thoughts? I'm thinking about

- [ ] `android.annotation.TargetApi` https://developer.android.com/reference/android/annotation/TargetApi
- [ ] `tools:targetApi` https://developer.android.com/studio/write/tool-attributes#toolstargetapi
- [ ] `androidx.annotation.RequiresApi`
- [ ] `androidx.annotation.ChecksSdkIntAtLeast`
- [ ] `androidx.test.filters.SdkSuppress`

and what they do, in what practical cases they might be useful?

**Comment by ebelinski** [comment]
I think this could be a good idea. Are you thinking of these being in a separate table alongside the other definitions tables? I don't have much familiarity with these annotations myself.

**Comment by TWiStErRob** [comment]
Yeah, somehow separately, as they're tighly related to API levels. A table at the bottom could be good. I can start the PR and then get some experienced community members to have a look and review/expand.

**Comment by TWiStErRob** [comment]
Yeah, somehow separately, as they're tighly related to API levels. A table at the bottom could be good. I can start the PR and then get some experienced community members to have a look and review/expand. In the end the format could change bu the contents will stay the same-ish.
